File: Examples/VisualBasic/Articles/ChartLegendEntry.vb
Imports Aspose.Cells
Imports Aspose.Cells.Charts

Namespace Articles
    Class ChartLegendEntry
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Open the template file.
            Dim workbook As New Workbook(dataDir & Convert.ToString("Sample.xlsx"))

            ' Access the first worksheet
            Dim sheet As Worksheet = workbook.Worksheets(0)

            ' Access the first chart inside the sheet
            Dim chart As Chart = sheet.Charts(0)

            ' Set text of second legend entry fill to none
            chart.Legend.LegendEntries(1).IsTextNoFill = True

            ' Save the workbook in xlsx format
            workbook.Save(dataDir & Convert.ToString("ChartLegendEntry_out_.xlsx"), SaveFormat.Xlsx)
            ' ExEnd:1
        End Sub
    End Class
End Namespace

File: Examples/CSharp/Articles/ChartLegendEntry.cs
using System;
using Aspose.Cells;
using Aspose.Cells.Charts;

namespace Aspose.Cells.Examples.CSharp.Articles
{
    class ChartLegendEntry
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            // Open the template file.
            Workbook workbook = new Workbook(sourceDir + "sampleChartLegendEntry.xlsx");

            // Access the first worksheet
            Worksheet sheet = workbook.Worksheets[0];

            // Access the first chart inside the sheet
            Chart chart = sheet.Charts[0];

            // Set text of second legend entry fill to none
            chart.Legend.LegendEntries[1].IsTextNoFill = true;

            // Save the workbook in xlsx format
            workbook.Save(outputDir + "outputChartLegendEntry.xlsx", SaveFormat.Xlsx);

            Console.WriteLine("ChartLegendEntry executed successfully.");
        }
    }
}
